containers/java/clipper-java-container/src/main/java/clipper/container/app/Model.java

```diff
@@ -28,8 +28,12 @@ public DataType getInputType() {
   }
 
   public DataType getOutputType() {
-    return DataType.Floats;
+    return DataType.Strings;
   }
 
-  public abstract FloatVector predict(I inputVector);
+  /**
+   * @return A JSON-formatted serializable string to be
+   * returned to Clipper as a prediction result
+   */
+  public abstract SerializableString predict(I inputVector);
```

Comment:
@Corey-Zumar Note that `Model.predict` should actually take the full batch of input vectors at once, rather than having the RPC system call `predict` in a for-loop. This allows models to apply batch processing optimizations if they want. E.g. this is particularly critical for the tensorflow model when running on a GPU.

I fixed this in the Python code but I'm leaving it as-is for now in the Java container code. I have a fix for this as part of my re-org of the Java codebase in #133.